scripts/pylib/twister/twisterlib/harness.py

```diff
@@ -318,53 +316,26 @@ def _generate_parameters_for_hardware(self, handler):
 
     def run_command(self, cmd, timeout):
         cmd, env = self._update_command_with_env_dependencies(cmd)
-        with subprocess.Popen(cmd,
-                              stdout=subprocess.PIPE,
-                              stderr=subprocess.STDOUT,
-                              env=env) as proc:
+        with subprocess.Popen(
+            cmd,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.STDOUT,
+            env=env
+        ) as proc:
             try:
                 reader_t = threading.Thread(target=self._output_reader, args=(proc,), daemon=True)
                 reader_t.start()
                 reader_t.join(timeout)
                 if reader_t.is_alive():
                     terminate_process(proc)
-                    logger.warning('Timeout has occurred.')
+                    logger.warning('Timeout has occurred. Can be extended in testspec file. '
+                                   f'Currently set to {timeout} seconds.')
```

Comment:
Please use `%` instead of `f-string` in logging message.

Reply:
For debug level logging it is better to use `%` instead of `f-string`, because of performance. But when using info, warning or higher level of logging, and there is not too much of such a logs it shouldn't be a problem